xcpretty/xcpretty.go

```diff
@@ -122,7 +122,9 @@ func (x xcpretty) IsInstalled() (bool, error) {
 		return false, err
 	}
 
-	return ruby.NewEnvironment(factory, locator).IsGemInstalled("xcpretty", "")
+	logger := log.NewLogger()
```

Comment:
To follow the patterns used, it would make more sense to add logger as a param in NewXcpretty() (even if it is a breaking change).

Reply:
updated!